web/src/compose_actions.ts

```diff
@@ -90,24 +109,28 @@ function clear_box() {
     $(".compose_control_button_container:has(.add-poll)").removeClass("disabled-on-hover");
 }
 
-export function autosize_message_content(opts) {
+let autosize_callback_opts: ComposeActionsStartOpts;
+export function autosize_message_content(opts: ComposeActionsStartOpts): void {
     if (!compose_ui.is_full_size()) {
-        autosize($("textarea#compose-textarea"), {
-            callback() {
-                maybe_scroll_up_selected_message(opts);
-            },
-        });
+        // TODO: the specifics of this will be handled in https://github.com/zulip/zulip/pull/29578
+        autosize_callback_opts = opts;
+        $("textarea#compose-textarea")
+            .off("autosize:resized")
+            .one("autosize:resized", () => {
+                maybe_scroll_up_selected_message(autosize_callback_opts);
+            });
+        autosize($("textarea#compose-textarea"));
```

Comment:
Is the intent that https://github.com/zulip/zulip/pull/29578 be merged first? It doesn't seem updated.

Reply:
Yes that's the intent, as per the code comment in this block.

I left a comment on that thread here: https://github.com/zulip/zulip/pull/29578#discussion_r1550659301

Reply:
I just also put it as a prep commit in this PR